crane/data.py

```diff
@@ -112,7 +112,7 @@ def load_all(app):
 
     try:
         data_dir = app.config[config.KEY_DATA_DIR]
-        logging.info('loading metadata from %s' % data_dir)
+        logger.info('loading metadata from %s' % data_dir)
```

Comment:
FYI: passing *data_dir* as an argument to info() instead of using % is how the Logger API is designed to be used.  It's also more efficient because the string variable substitution is only performed when the logging level and other criteria is met.  This is more important for debug() though.